pkg/virt-handler/cmd-client/client.go

```diff
@@ -162,13 +162,11 @@ func IsSocketUnresponsive(socket string) bool {
 }
 
 func MarkSocketUnresponsive(socket string) error {
-	file := filepath.Join(filepath.Dir(socket), StandardLauncherUnresponsiveFileName)
-	f, err := os.Create(file)
+	dir, err := safepath.NewPathNoFollow(filepath.Dir(socket))
 	if err != nil {
 		return err
 	}
-	f.Close()
-	return nil
+	return safepath.TouchAtNoFollow(dir, StandardLauncherUnresponsiveFileName, 0666)
```

Comment:
Reading the documentation of `TouchAtNoFollow`:
> If the target file exists already, the function will fail.

This is a small change of behavior, since `os.Create(file)` does not fail in that case.
Should we check the error and ignore a "the file already exist" error?